.github/workflows/publish.yml

```diff
@@ -22,14 +21,36 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - run: npm ci
+      # Bump the version to the current
+      - name: 'Setting version of libraries to ${{ github.ref_name }} for libraries located in libraries/'
+        run: npm version ${{ github.ref_name }} -w libraries/ --force --no-git-tag-version
 
-      - run: npm run build
+      - name: 'Rebuilding affected libraries'
+        run: npm run build -w libraries/
 
-      - run: npm run publish-lib:dry
+      - name: 'Commit changes to master branch'
+        run: |
+          git config user.name "Six Webcomponents Release Bot"
+          git config user.email "<>"
+          git add -A
```

Comment:
might be safer here if we just add package.json and package-lock.json

Reply:
Ok